### PR TITLE
Franred/provisioning vm with omnibus to build flang

### PR DESCRIPTION
**Pull request by franred**

### CODE REVIEW

File: flang-vm.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+- hosts: GCCVM
+  tasks:
+  - name: Install epel-repository
```
Comment (lukaszkolodziejczyk) [inline flang-vm.yml]: Should be: "Install epel-repository and centos-release-scl" or "Install repositories"

**Comment by lukaszkolodziejczyk** [comment]
I've done the following to check the provisioning script:
- Build LLVM -- works, check-llvm -- sucess
- Build Clang -- works, check-clang -- sucess, compile and run "hello_world.c" -- sucess
- Build Flang -- works, check-flang -- sucess, compile and run "hello_world.f90" -- failed

Error: 
```
./a.out: error while loading shared libraries: libflang.so: cannot open shared object file: No such file or directory
```
Solution: 
`export LD_LIBRARY_PATH=$INSTALLDIR/lib:$LD_LIBRARY_PATH` where `INSTALLDIR=/opt/codethink-flang`

After applying the solution, everything works fine.

**Comment by franred** [comment]
I fixed both comments and force pushed this branch.

**Comment by lukaszkolodziejczyk** [comment]
This works correctly, I'm merging the pull request.

**Pull request merged**